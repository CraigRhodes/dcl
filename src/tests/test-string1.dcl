int hamilton() {
	int thomasjefferson = 1789;
	bool myshot = true;
	return thomasjefferson;
}

string colorpurple() {	
	string leadin = "Like a blade of corn, like a honey bee";
	string color = "Like the color purple. Where do it come from?";
	string response = leadin + color;
	print_string(response);
	return response
	
}

double groundhog_day() {
	double dayone = 42.0;
	double daytwo = 42.0;
	double daythree = 42.0;
	print_string("it's groundhog day!!!!!!!")
	return daythree; 


}

int main(){
	int ham = hamilton();
	colorpurple();
	groundhog_day();
	print_int(ham);
	return ham;

	
}
//verifies string concatenation 

