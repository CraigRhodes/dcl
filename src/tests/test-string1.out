Like a blade of corn, like a honey bee Like the color purple. Where do it come from?
it's groundhog day!!!!!!!
1789